mutation/mutate.go

```diff
@@ -204,15 +212,21 @@ func addAutoTraceLabel(objectMeta *metav1.ObjectMeta, value string) {
 	}
 }
 
-func (m *mutatorImpl) removeLumigoFrom(topLevelObjectMeta *metav1.ObjectMeta, podTemplateSpec *corev1.PodTemplateSpec) error {
+func (m *mutatorImpl) removeLumigoFrom(topLevelObjectMeta *metav1.ObjectMeta, podTemplateSpec *corev1.PodTemplateSpec) (bool, error) {
+	originalSpec := podTemplateSpec.Spec.DeepCopy()
+
 	if err := m.removeLumigoFromPodSpec(&podTemplateSpec.Spec); err != nil {
-		return err
+		return false, err
+	}
+
+	if reflect.DeepEqual(originalSpec, &podTemplateSpec.Spec) {
+		return false, nil
```

Comment:
maybe give some kind of an error here? "remove didn't do anything?"

Reply:
Yes, good point. Done in 58b56e5.